src/usePlaidLink.ts

```diff
@@ -86,7 +73,18 @@ export const usePlaidLink = (options: PlaidLinkOptions) => {
     products,
   ]);
 
-  const ready = plaid != null && (!loading || iframeLoaded);
+  useEffect(() => {
+    setIframeLoaded(false);
+    if (!plaid) {
+      return;
+    }
+    return () => {
+      // destroy the Plaid iframe factory when unmounting an instance
+      plaid.exit({ force: true }, () => plaid.destroy());
+    }
+  }, [plaid])
+
+  const ready = !!plaid && !loading && iframeLoaded; 
```

Comment:
This was previously

```ts
const ready = plaid != null && (!loading || iframeLoaded);
```

But reading that code, it seems to me that `iframeLoaded` is completely unused - it can't be `true` without being first `loading === true`, so I'm guessing this fixes some edge-case bug